Q_acc_x: 0.0001
Q_acc_y: 0.0001
Q_acc_z: 0.0001
Q_angular_x: 0.00001
Q_angular_y: 0.00001
Q_angular_z: 0.00001

outlier_constant: 1.5

pose_p_cov: 0.01
vel_p_cov: 0.001
angle_p_cov: 0.01

p_drift_p_cov: 0.0001
q_drift_p_cov: 0.0001

use_cam_imu: 1

estimate_acc_bias: 1
acc_bias_var: [ 0.001,0.001,0.001]
acc_bias_p_cov: 0.001

estimate_gyro_bias: 1
gyro_bias_var: [ 0.001,0.001,0.001]
gyro_bias_p_cov: 0.001

estimate_gravity: 1
g: [0,0,-10.34]
g_p_cov: 0.1

#position sensor params
Sensor_prefix: [camera,pozyx]
#pozyx
pozyx_R_pose: [1, 1, 3]
pozyx_is_orientation_sensor: 0
pozyx_estimate_drift: 0
pozyx_topic: /pozyx/measured
pozyx_msg_type: 1
pozyx_rotation: [0.0, 1.0, 0.0,1.0, 0.0, 0,0, 0.0, 1.0]
pozyx_translation: [-5.071980,-3.987154,-0.2]
#camera
camera_is_orientation_sensor: 1
camera_topic: /camera/odom/sample
camera_msg_type: 0
camera_R_pose: [0.05, 0.05, 0.01]
camera_R_angle: [0.01, 0.01, 0.01, 0.01]
camera_estimate_drift: 1
camera_rotation: [1,0,0,0,1,0,0,0,1]
camera_translation: [0,0,0]
